<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SOFilterDebugerClass</class>
 <widget class="QMainWindow" name="SOFilterDebugerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>914</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SOFilterDebuger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QOpenGLWidget" name="openGLWidget"/>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="ControlGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string>Control</string>
      </property>
      <widget class="QPushButton" name="tickButton">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Tick</string>
       </property>
      </widget>
      <widget class="QPushButton" name="RollBackButton">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>RollBack</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="SpeedControlGroup">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>10</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="title">
        <string>Speed</string>
       </property>
       <widget class="QDoubleSpinBox" name="speedBox">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="stopButton">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
      <widget class="QPushButton" name="startButton">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SOFilterDebuger.qrc"/>
 </resources>
 <connections/>
</ui>
